components/header.tsx

```diff
@@ -1,32 +1,20 @@
 import Image from 'next/image';
+import { Group } from '@mantine/core';
+
 import logo from '../public/aclima-logo.svg';
-import theme from '../utils/theme';
+import classes from './header.module.css';
```

Comment:
seems like we do have to switch to css modules then?

Reply:
where is the `header.module.css` file?

Reply:
gone, let me fix that line